### DISCUSSION TITLE
Can a noise filter be added when upscaling older media?

### DISCUSSION THREAD

**Discussion by UsaNekoNetizen** [bloc97/Anime4K#254] [Ideas]
I'm new to github and everything of the sort so forgive my questions. 
I was testing out the program on an anime from the 90s based on it's region 2 dvd master. And I shared some screenshots with a buddy testing out all of the filters (my preference is using a mix of Modes B, B+B and C+A). 

As my friend directly points out:
"I think they need a noise layer added for extra cohesion. When I do [my] upscales and clean up cover art I usually layer the original low res version with a noise layer on top of the upscale to get rid of the smear artefacts and overly clean areas you get with AI upscaling."
Original
![mpv-shot0007](https://github.com/user-attachments/assets/6aaf2630-ad8d-4473-85a3-ca4cc61a1fd2)
Mode B+B
![mpv-shot0011](https://github.com/user-attachments/assets/b639c06b-a8fd-4ddb-8925-fecbf2587f57)
Mode C + A
![mpv-shot0010](https://github.com/user-attachments/assets/14762543-48ca-4696-98e9-5605b1559c6c)

**Comment by Tama47** [comment]
Anime4K is not very good with noises or added grains in anime. I recommend giving these shaders a try, and see if you like them better for older anime with noises.

[AnimeJaNai V3](https://github.com/the-database/mpv-upscale-2x_animejanai)
[ArtCNN](https://github.com/Artoriuz/ArtCNN)
[CuNNy](https://github.com/funnyplanter/CuNNy)
